charge.go

```diff
@@ -91,6 +92,15 @@ type FraudDetails struct {
 	StripeReport FraudReport `json:"stripe_report"`
 }
 
+// Outcome is the charge's outcome that details whether a payment
+// was accepted and why.
+type Outcome struct {
```

Comment:
(And to be clear, I'm suggesting the change because `Outcome` lives in the parent namespace which increases the likelihood of some eventually collision.)